src/Fridge.java

```diff
@@ -0,0 +1,57 @@
+import components.map.Map;
+import components.map.Map1L;
+
+public class Fridge {
+    private boolean isOpen;
+    private static Map<String, Integer> fridgeItems;
+    private Map<String, Integer> freezerItems;
+
+    public Fridge() {
+        this.isOpen = false;
+        this.fridgeItems = new Map1L<>();
+        this.freezerItems = new Map1L<>();
+    }
+
+    public void addOneItem(String item, boolean isOpen,
```

Comment:
What is the location in this case? Would it be reasonable to hide the concept of the map from the user? I think so. I would instead ask for a boolean here (true for fridge; false for freezer). If you won't to overengineer things a bit, you could make an enum, so they user could supply something like `Fridge.Fridge` or `Fridge.Freezer`. Or, you could just make two different methods to avoid this problem entirely (i.e., `addItemToFridge` and `addItemToFreezer`).